reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -833,12 +833,14 @@
   } ],
   "vulnerabilities_resolutions" : [ ],
   "vulnerabilities" : [ {
+    "_id" : 0,
```

Comment:
This commit should be squashed with the previous commit following the principle that every single commit of a PR should pass the tests.

Reply:
Squashed the commit